Tools/WinMLRunner/BindingUtilities.h

```diff
@@ -1,139 +1,58 @@
 #pragma once
+#include <random>
 #include "Common.h"
 #include "ModelBinding.h"
-using namespace winrt::Windows::Graphics::Imaging;
+#include "CommandLineArgs.h"
+
 using namespace Windows::Media;
-using namespace winrt::Windows::Foundation::Collections;
 using namespace Windows::Storage;
 using namespace winrt::Windows::AI::MachineLearning;
+using namespace winrt::Windows::Foundation;
+using namespace winrt::Windows::Foundation::Collections;
+    using namespace winrt::Windows::Graphics::DirectX;
+    using namespace winrt::Windows::Graphics::Imaging;
 
 namespace BindingUtilities
 {
-    void BindTensorsFromGarbageData(LearningModelBinding context, LearningModel model)
```

Comment:
Bing refactoring here. Instead of using this huge switch statement at 2 different places, I place it in a single method called by both garbage data and CSV data.